src/lib/components/prompt/PromptSuggestion.svelte

```diff
@@ -80,10 +80,10 @@
 			{/if}
 		{/if}
 	</div>
-	<div class="mt-2 flex items-center justify-end gap-4">
+	<div class="mt-2 flex items-center justify-end gap-3">
 		{#if !applied && !applyingSuggestion}
 			<button
-				class="cursor-pointer text-gray-inactive transition {!disabled
+				class="cursor-pointer text-sm text-gray-active transition {!disabled
```

Comment:
we disagree on this but I think the disabled color is too light for this. I made the text smaller instead

Reply:
lgtm like this